tests/bsim/bluetooth/host/adv/periodic/src/per_adv_sync.c

```diff
@@ -190,11 +190,19 @@ static void start_scan(void)
 	int err;
 
 	printk("Start scanning...");
+#if defined(CONFIG_BT_CTLR_PHY_CODED)
```

Comment:
Consider using `IS_ENABLED` instead where applicable as per the coding guidelines

Reply:
Yes. Consider replacing the preprocessor `if` with a variable `params = IS_ENABLED(CONFIG_BT_CTLR_PHY_CODED) ? BT_LE_SCAN_CODED_ACTIVE : BT_LE_SCAN_ACTIVE;`